cmd/wallet-sdk-gomobile/openid4vp/interaction.go

```diff
@@ -39,15 +40,29 @@ type Interaction struct {
 	goAPIOpenID4VP   goAPIOpenID4VP
 	didResolver      api.DIDResolver
 	inquirer         *credential.Inquirer
+	oTel             *otel.Trace
 }
 
 // NewInteraction creates a new OpenID4VP Interaction.
 // The methods defined on this object are used to help guide the calling code through the OpenID4CI flow.
-func NewInteraction(args *Args, opts *Opts) *Interaction {
+func NewInteraction(args *Args, opts *Opts) (*Interaction, error) { //nolint:funlen
```

Comment:
Need to update `usage.md` to show the error being passed in in the Swift examples (you can follow the pattern from the other Swift examples)

Also this is a breaking change, so this PR should have an `!` in it (i.e. `feat(sdk)!`)